### ISSUE TITLE
ESP32­S2­WROOM Not recommended for new designs (NRND)

### ISSUE THREAD

**Issue by iflyhere** [DanD222/sensorboard_NewHardware#4]
Maybe it's possible to switch to a ESP32 C3 WROOM 02 for bluetooth compatibility too.

**Comment by DanD222** [comment]
Yes - that's the plan, along with other changes suggested by @hpmax, I just need to get round to implementing them :(

**Comment by hpmax** [comment]
ESP32-S3, not C3 please.  I think S3 is Xtensa, and C3 is RISC-V.

1) There are a number of other changes that I'd like.  I think some of the interrupt lines which we haven't been using should be routed back to the ESP32. 

2) The LPS22HH should be replaced by LPS22DF (fortunately this is pin compatible and just minor software changes).

3) The LSM6DSM should be replaced by LSM6DSR (again, I think this is just a software change -- but it is a more extensive software change).

Alternately, Kris Winer seems to now be recommending the ICM42688P, which has slightly worse accelerometer performance but MUCH better gyro performance -- I think this is because the gyros require an accurate clock, and the ICM42688 can use an external crystal reference and the LSM6 series cannot.  This would be a more significant change, more research is needed.

Right now I'm struggling with the software.  First, what filtering/sensor fusion algorithm to use to make use of the the 9DOF data, and second I've never written ESP32 code before... hopefully that's more straightforward.
